src/main/java/se/iths/SpringbootGroupProject/entities/Message.java

```diff
@@ -18,8 +18,14 @@ public class Message {
 
     private String title;
     private String messageBody;
-    private boolean isPrivateMessage;
+    private boolean privateMessage;
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "user_id")
+    private User user;
 
+    public User getUser() {
+        return user;
+    }
```

Comment:
I did it because we would not want to change user on a message. I did not think that through though. But we do need a setter to set a user for a created message ofc. I pushed an update.